compiler/luci-interpreter/src/loader/nodes/Conv2D.cpp

```diff
@@ -38,6 +39,18 @@ std::unique_ptr<Kernel> build_kernel_CircleConv2D(const luci::CircleNode *circle
     std::make_unique<Tensor>(input->element_type(), Shape({}), AffineQuantization{}, "");
   im2col->set_observable(false);
   im2col->set_data_buffer(nullptr);
+  // If node has execution plan then read memory offsets for im2col temporary tensor
+  // from the beginning of shared memory buffer.
+  // Used in Static Memory Manager.
+  // TODO move tensors offset initialization to one place
+  if (luci::has_execution_plan(node))
+  {
+    auto memory_plan = luci::get_execution_plan(node);
```

Comment:
(optional)  May be const?

Reply:
Done, and also rename old naming `memory_plan` to `execution_plan`